### DISCUSSION TITLE
Задержка при прослушивании двух диапазонов.

### DISCUSSION THREAD

**Discussion by ua9sup** [egzumer/uv-k5-firmware-custom#346] [General]
В баофенге uv-5r прослушивание двойной частоты происходит без задержек,а на кваншенге uv-k5r плюс,задержка 4секунды примерно,что совсем неудобно. У меня домашний кросс бенд,стоит двух диапазонный трансивер и антенна на высоте 19 метров,общение 145.500 на 435.000 и обратно. Все было хорошо с баофенгом,пока в руки не попал кваншенг. Прошивки разные ставил,не помогает. Может кто сталкивался с такой проблемой?

**Comment by ErikS-web** [comment]
Please use English.

Google translate: 
**Delay when listening to two bands.**
_WOULD BE NICE TO ADD SSB TO THE DEMODU In the baofeng uv-5r, listening to the double frequency occurs without delay, but in the quangsheng uv-k5r plus, the delay is approximately 4 seconds, which is completely inconvenient. I have a home cross-band, there is a two-band transceiver and an antenna at an altitude of 19 meters, communication is 145,500 to 435,000 and back. Everything was fine with the baofeng until the quangsheng fell into my hands. I installed different firmware, it doesn’t help. Has anyone encountered this problem?_

It that right... type uv-k**R**5 ?

What do you mean with "different firmware". Here we use and discuss release from [EGZUMER](https://github.com/egzumer/uv-k5-firmware-custom/releases).

**Reply by ua9sup** [reply]
Была прошивка сток,а поставил Вашу v021. Везде происходит задержка.Спасибо!

**Reply by ErikS-web** [reply]
Please use English lanquage

**Reply by ua9sup** [reply]
Я его не знаю.

**Reply by ua9sup** [reply]
На Русский язык Вы не отвечаете?

**Reply by OE1MWW** [reply]
No, we use the english language in this Github

Нет, мы используем английский язык
Сформулируйте свою проблему и воспользуйтесь Google translate. Ваш первый запрос был непонятен.

**Reply by OE1MWW** [reply]
используйте translate.google.com для чтения

4 seconds delay ??? this is not possible in the UV-K5, because this would be 'store-and-forward'. The UV-K5 does not have any memory to store speech. Max. delay I have seen by SDR software is around 300 mSec. If it's 4 seconds (!) you have to check what you receive.

**Comment by ua9sup** [comment]
При переключении происходит задержка. С диапазона на диапазон,для работы в домашнем крос бенде.
--
Отправлено из Mail.ru для Android пятница, 29 декабря 2023г., 16:57 +05:00 от OE1MWW  ***@***.*** :

**Comment by ua9sup** [comment]
При работе с баофенг uv-5r такого нет. Все переключается быстро.
--
Отправлено из Mail.ru для Android пятница, 29 декабря 2023г., 16:57 +05:00 от OE1MWW  ***@***.*** :

**Reply by OE1MWW** [reply]
a.) we told you to use english
b.) your inital request is still not understandable
how is your setup connected ? what device is receiver, what device is transmitter ?
we don't care how high your antenna is. please stay with the relevant facts.

**Comment by ua9sup** [comment]
OK!
--
Отправлено из Mail.ru для Android пятница, 29 декабря 2023г., 18:48 +05:00 от OE1MWW  ***@***.*** :
